src/Raven.Server/ServerWide/Commands/Subscriptions/AcknowledgeSubscriptionBatchCommand.cs

```diff
@@ -21,7 +20,10 @@ namespace Raven.Server.ServerWide.Commands.Subscriptions
     public class AcknowledgeSubscriptionBatchCommand : UpdateValueForDatabaseCommand
     {
         public string ChangeVector;
-        public string LastKnownSubscriptionChangeVector; // Now this used only for backward compatibility
+
+        // in regular subscription LastKnownSubscriptionChangeVector used only for backward compatibility
+        // in sharing subscription LastKnownSubscriptionChangeVector used to save orchestrator cv
+        public string LastKnownSubscriptionChangeVector; 
```

Comment:
what we will have here then for orchestrator vs ChangeVector field?

Reply:
for shard it will be:
`ChangeVector` = local change vector for shard
 `LastKnownSubscriptionChangeVector` = merged change vector of the orchestrator
 
 both will be persisted in this command in to the `ShardingState` class